packages/ui/src/atom/link-atom/link-atom.theme-default.scss

```diff
@@ -15,15 +15,15 @@
   }
 }
 
-.v-link.theme-default.is-underline {
+.link-atom.theme-default.is-underline {
   text-decoration: underline;
   &:focus,
   &:hover {
     text-decoration: none;
   }
 }
 
-.v-link.theme-default[disabled] {
+.link-atom.theme-default[disabled] {
```

Comment:
`disabled` is not semantically right for `a`. I think it should `is-disabled` class on link level.

Reply:
hmm, it's not related for current task